content/en/blog/general/lets-play-kyverno/index.md

```diff
@@ -0,0 +1,95 @@
+
+![Kyverno Playground: Share Dialog](share.png)
+
+### Context (Variables)
+
+Kyverno can run on various operations and uses request information such as usernames, groups, roles and additional context from provided ConfigMaps, Secrets resources and more. How could we provide this information within the Playground? We don't have a true cluster request or other resources to pull the information from. We decided to take a simple approach and provide the ability to simulate what you need to provision for your policy.
```

Comment:
"...such as usernames, groups, roles, and additional context from provided ConfigMaps, Secrets, and more."